crowbar_framework/lib/remote_node.rb

```diff
@@ -143,6 +143,14 @@ def self.ssh_cmd(host_or_ip, cmd)
     system(*ssh)
   end
 
+  def self.pipe_ssh_cmd(host_or_ip, cmd)
+    ssh = self.ssh_cmd_base(host_or_ip)
+    ssh << cmd
+    IO.popen(ssh, 'w+') do |pipe|
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.